### ISSUE TITLE
jupyterlab not working, though the install proceeded without complaint

### ISSUE THREAD

**Issue by philipobrien** [sdispater/poetry#1292]
- [x] I am on the [latest](https://github.com/sdispater/poetry/releases/latest) Poetry version.
- [x] I have searched the [issues](https://github.com/sdispater/poetry/issues) of this repo and believe that this is not a duplicate.
- [x] If an exception occurs when executing a command, I executed it again in debug mode (`-vvv` option).

- **OS version and name**: macOS Mojave 10.14.6
- **Poetry version**: 0.12.17
- **Link of a [Gist](https://gist.github.com/) with the contents of your pyproject.toml file**: https://gist.github.com/philipobrien/3153875e64cee2a87d00918ac8e734cb

## Issue
I have just tried out poetry for the first time, having been using pipenv previously. Unfortunately I have run straight into an issue that wasn't occuring when performing the same steps with pipenv. 

Basically, I installed [jupyter lab](https://github.com/jupyterlab/jupyterlab), the install proceeded without issue, but when attempting to kickstart it using `jupyter lab` from within the poetry shell I'm getting

```
[C 16:19:57.384 LabApp] Bad config encountered during initialization:
[C 16:19:57.384 LabApp] The 'contents_manager_class' trait of <jupyterlab.labapp.LabApp object at 0x1091af2d0> instance must be a type, but 'jupytext.TextFileContentsManager' could not be imported
```

I'm not seeing this at all with pipenv so I'm presuming it's poetry related, though I donb't even know how to go about confirming that.

**Comment by philipobrien** [comment]
Apologies, you can close this. I have discovered that it is nothing to do with poetry. There was a missing install of the `jupytext` package that my global `jupyter` config was expecting. All it took was a `poetry add jupytext` and it works perfectly.
